app/assets/javascripts/wpcc/components/SalaryConditions.js

```diff
@@ -95,24 +78,24 @@ define(['jquery', 'DoughBaseComponent'], function($, DoughBaseComponent) {
   }
 
   SalaryConditions.prototype._belowManualOptIn = function(salary, frequency) {
-    var thresholds = optInTriggers[frequency];
+    var thresholds = this.optInTriggers[frequency];
```

Comment:
What's the difference between `optInTriggers[frequency]` and `this. optInTriggers[frequency]`? Is it necessary to precede it with `this.` contrary to what had originally been done by the _Frontend_ team, please?

Reply:
Before the variable was on local scope of the salary conditions file and now it is an *attribute* of the object salary conditions which is the `this` above and in all the other changes. I recommend to take a look on this doc: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this

Reply:
I added the explanation above on the commit message as well.